### ISSUE TITLE
Start up page 'get started' button is in incorrect alignment.

### ISSUE THREAD

**Issue by MadhushaPrasad** [MadhushaPrasad/Diagon-Alley#85]
Correct UI :
![Startup](https://github.com/user-attachments/assets/e677e797-f9eb-4009-b342-d2a36c50f7f3)

Incorrect UI alignment:
<img width="1011" alt="Screenshot 2024-10-06 at 20 22 20" src="https://github.com/user-attachments/assets/458f41d0-2aa2-4f60-8c1e-809a406f6359">

**Comment by aayu5hgit** [comment]
Hey @MadhushaPrasad 

I'll be happy to help you fix this bug.

**Comment by MadhushaPrasad** [comment]
> Hey @MadhushaPrasad
> 
> I'll be happy to help you fix this bug.

@aayu5hgit sure go ahead

**Comment by aayu5hgit** [comment]
Will he raising a PR by tonight

**Comment by MadhushaPrasad** [comment]
> Will he raising a PR by tonight

no, you are the only one assign for this issue, so you can work on this issue

**Comment by aayu5hgit** [comment]
Cool, I've already started.

**Comment by MadhushaPrasad** [comment]
@aayu5hgit any update?

**Comment by parrth20** [comment]
once this PR is merged I'll work on other issue
